source/javascripts/lib/wirsing/filters/search.coffee

```diff
@@ -0,0 +1,21 @@
+#= require ../module
+
+# Returns a configurable search function that can be used
+# to filter an array of objects by searching for a term
+# within the specified property of each object.
+
+wirsing.module 'filter.search', ->
+
+  (options) ->
+
+    search = (items) ->
+      if search.term.length < search.minChars then return items
+      else items.filter (item) -> item[search.in].search(search.term) > -1
```

Comment:
Ah, you didn't understand it completely :-) it is not necessary to call `search.configure` – this is done once automatically during the creation of the filter (last line of code). This also sets the default params. I just reused the `search.configure` method during the initialization. Look at the specs, there you see that it works correctly.